__tests__/index.test.ts

```diff
@@ -311,6 +313,27 @@ describe('Auth0', () => {
         connection: 'test-connection'
       });
     });
+    it('creates correct query params when overriding redirect_uri', async () => {
+      const redirect_uri = 'https://custom-redirect-uri/callback';
+      const { auth0, utils } = await setup({
+        redirect_uri
+      });
+
+      await auth0.loginWithRedirect(REDIRECT_OPTIONS);
+
+      expect(utils.createQueryParams).toHaveBeenCalledWith({
+        client_id: TEST_CLIENT_ID,
+        scope: TEST_SCOPES,
+        response_type: TEST_CODE,
+        response_mode: 'query',
+        state: TEST_ENCODED_STATE,
+        nonce: TEST_RANDOM_STRING,
+        redirect_uri: REDIRECT_OPTIONS.redirect_uri,
```

Comment:
shouldn't this be `redirect_uri: redirect_uri` ?

Reply:
@lbalmaceda I don't think so. The test determines that the options passed into `loginWithRedirect` take precedence over the options passed when creating the client. `redirect_uri` here represents the options passed into the client.